cmd/otelcontribcol/go.mod

```diff
@@ -4,8 +4,6 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/cmd/otelcontrib
 
 go 1.21.0
 
-toolchain go1.21.10
-
```

Comment:
I believe we don't want to delete this, and I think dropping this causes the `make genotelcontribcol` failure.

Reply:
no apologies needed, you were right :)